opencga-app/app/scripts/docker/opencga-init/init-config.py

```diff
@@ -0,0 +1,72 @@
+import oyaml as yaml
+import argparse
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--config-path", help="path to the configuration.yml file", default="/opt/opencga/conf/configuration.yml")
+parser.add_argument("--client-config-path", help="path to the client-configuration.yml file", default="/opt/opencga/conf/client-configuration.yml")
+parser.add_argument("--storage-config-path", help="path to the storage-configuration.yml file", default="/opt/opencga/conf/storage-configuration.yml")
+parser.add_argument("--search-host", required=True)
+parser.add_argument("--clinical-host", required=True)
+parser.add_argument("--catalog-database-host", required=True)
+parser.add_argument("--catalog-database-user", required=True)
+parser.add_argument("--catalog-database-password", required=True)
+parser.add_argument("--catalog-search-host", required=True)
+parser.add_argument("--catalog-search-user", required=True)
+parser.add_argument("--catalog-search-password", required=True)
+parser.add_argument("--rest-host", required=True)
+parser.add_argument("--grpc-host", required=True)
+parser.add_argument("--save", help="save update to source configuration files (default: false)", default=False, action='store_true')
+args = parser.parse_args()
+
+# Load storage configuration yaml
+with open(args.storage_config_path) as f:
+    storage_config = yaml.load(f)
+
+# Inject search and clinical hosts
+storage_config["search"]["host"] = args.search_host
+storage_config["clinical"]["host"] = args.clinical_host
+
+# Load configuration yaml
+with open(args.config_path) as f:
+    config = yaml.load(f)
+
+# Inject catalog database
+config["catalog"]["database"]["hosts"][0] = args.catalog_database_host
```

Comment:
Add support for multiple hosts

Reply:
Done in commit (https://github.com/opencb/opencga/pull/969/commits/166f39594a0f7d55faf09583c087fd4ead965ba5)